site/tutorials/keptn-litmus-018/codelab.json

```diff
@@ -0,0 +1,34 @@
+{
+  "environment": "web",
+  "format": "html",
+  "prefix": "https://storage.googleapis.com",
+  "mainga": "UA-133584243-1",
+  "updated": "2022-12-05T16:29:24+01:00",
+  "id": "keptn-litmus-018",
+  "duration": 44,
+  "title": "Resilience evaluation with LitmusChaos, Prometheus, and Keptn",
+  "authors": "Jürgen Etzlstorfer",
```

Comment:
I think you can add your name here ;)

Reply:
changed

Reply:
I missed this!  Shouldn't this show Shubham Chaudhary and karthik_s -- don't have an actual name for the second name?  They are the ones who stepped up and did all the work!

Reply:
did they rewrite it, or did they change the initial setup, I added shubham Chaudhary, but should i only write `karthik_s` or is there a full name available :)